src/js/components/DeploymentsListComponent.jsx

```diff
@@ -184,6 +191,11 @@ var DeploymentListComponent = React.createClass({
               Error fetching deployments. Refresh to try again.
             </td>
           </tr>
+          <tr className={unauthorizedClassSet}>
+            <td className="text-center text-danger" colSpan="6">
+              Error fetching deployments. Unauthorized access.
```

Comment:
That's an approach of hiding elements, that we've introduced in the past, it should let React render faster, because of better diffing. We are using this a lot in other places too, but I agree that this is strange and like to rediscuss this. Outside of this PR..